### DISCUSSION TITLE
How to load the pagination after a filter search query to the user

### DISCUSSION THREAD

**Discussion by AaronRiboy** [serenity-is/Serenity#6649] [Help]
Hello, i want to load the pagination after filter query because the related database table have 60,000 records. So its taking too much time and giving connection timeout. Even in sql server its takes time to load. Im trying to add filter button to the user before load the pagination to overcome this situation. Im still new to this framework. Hope someone will help me to guide . Thank you in advance

**Comment by wezmag** [comment]
In Grid.ts, you can override `getGridCanLoad`.

```
protected getGridCanLoad(): boolean
{
   if (filter is selected) 
      return true;
   else
     return false;
}
```
